src/03-data-analysis-in-r/04-transform-1.Rmd

```diff
@@ -181,7 +181,7 @@ mutate(FAES,
 
 A Couple of things to note: 
 
-  1. The variable we're creating needs to be in quotation marks, hence `dil_fct` for our dilution factor variable
+  1. Quotation marks are optional: When creating a new variable in mutate(), you can choose to use or not use quotation marks around the variable name. Both `dil_fct` and `"dil_fct"` will work, but it's more common to omit the quotation marks.
```

Comment:
This edit isn't technically true; I believe quotes are necessary when the variable contains spaces or special characters.